azure-pipelines.yml

```diff
@@ -0,0 +1,115 @@
+---
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+    - 1-stable
+pr:
+  autoCancel: true
+  branches:
+    include: 
+      - master
+      - 1-stable
+
+variables:
+  CHEF_LICENSE: accept-no-persist
+
+stages:
+  - stage: Build
+    jobs:
+      - job: Validate
+        strategy:
+          matrix:
+            Windows_Ruby25:
+              version: 2.5.3
+              imageName: 'windows-2019'
+              task: windows_ci.bat
+            Windows_Ruby26:
+              version: 2.6.1
+              imageName: 'windows-2019'
+              task: windows_ci.bat
+            Linux_Ruby25:
+              version: 2.5.5
+              imageName: 'ubuntu-16.04'
+              task: linux_ci.sh
+              KITCHEN_YAML: kitchen.dokken.yml
+            Linux_Ruby26:
+              version: 2.6.2
+              imageName: 'ubuntu-16.04'
+              task: linux_ci.sh
+              KITCHEN_YAML: kitchen.dokken.yml
+            Mac_Ruby25:
+              version: 2.5.5
+              imageName: 'macOS-10.13'
+              task: mac_ci.sh
+            Mac_Ruby26:
+              version: 2.6.2
+              imageName: 'macOS-10.13'
+              task: mac_ci.sh
+            Windows_Integration:
+              version: 2.6.1
+              imageName: 'windows-2019'
+              task: windows_integration.bat
+              machine_user: test_user
+              machine_pass: Pass@word1
+              machine_port: 5985
+              KITCHEN_YAML: kitchen.appveyor.yml
+            Proxy_Integration:
+              version: 2.5.5
+              imageName: 'ubuntu-16.04'
+              task: proxy_ci.sh
+              MACHINE_USER: kitchen
+              MACHINE_PASS: K1tch3nY@ml!
+              MACHINE_PORT: 22
+              KITCHEN_YAML: ../../../../../../../kitchen.proxy.yml
```

Comment:
Is there a more descriptive way of describing this path? (presume the root of the repo - so $(Build.SourcesDirectory) ?

Reply:
I'll try that - I tried an absolute path before, but I don't think kitchen liked it.